eagleproject/core/blockchain/harvest/transactions.py

```diff
@@ -125,6 +126,36 @@ def maybe_store_transfer_record(log, block):
 
     return transfer
 
+def maybe_store_wrap_transfer_record(log, block):
```

Comment:
Pretty cool that you are also storing WOUSD transafers. This will surely be useful for other use cases